### ISSUE TITLE
[preview-envs/observability] Prometheus CRD fails to install

### ISSUE THREAD

**Issue by ArthurSens** [gitpod-io/gitpod#6455]
Monitoring satellite fails 100% of the time right now because it fails to apply the Prometheus Custom Resource Definition into the cluster, with the following error:
```
The CustomResourceDefinition "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

**Comment by jithinjks** [comment]
is this solved, am stuck with this issue.

`g708ltrv@g708ltrv-Latitude-5400:~/Documents/workspace/deployment-artifactory/monitoring/production/new_bundle$ k apply -f bundle.yaml 
customresourcedefinition.apiextensions.k8s.io/alertmanagerconfigs.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/alertmanagers.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/probes.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/prometheusrules.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/servicemonitors.monitoring.coreos.com created
customresourcedefinition.apiextensions.k8s.io/thanosrulers.monitoring.coreos.com created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-operator created
clusterrole.rbac.authorization.k8s.io/prometheus-operator created
deployment.apps/prometheus-operator created
serviceaccount/prometheus-operator created
service/prometheus-operator created
The CustomResourceDefinition "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
`

**Comment by ArthurSens** [comment]
hey @jithinjks are you asking about the issue in this repository or about installing the CRDs?

BThe problem on our preview environments will be solved soon by #6458. but if you've come here to understand the CRD problems, just use `kubectl create -f` instead of `kubectl apply -f` and it should work
